README.md

```diff
@@ -86,7 +86,150 @@ There are several ways to execute the code cells in your notebook:
     panel. Here you can schedule your notebook to be executed once at some future
     time, or repeatedly at your specified interval.
 
-## 4. Save and Share
+## 4. Analyze the Results
+
+After running each cell of the notebook, the results will display. When we use pixiedust display() to create an interactive dataset, we are able to manipulate the data using tables, graphs, and charts.
```

Comment:
s/pixiedust/PixieDust/ ???  I think the right name is camel-case, but if you are referring to the import it is lower. Could go either way here (I'd go with camel because notebook just says display() but up to you.

another nit:  "manipulate the data" sounds to me like changing numbers. I dunno... 'change the visualization' ?